pages/linux/numactl.md

```diff
@@ -0,0 +1,15 @@
+# numactl
+
+> Control NUMA policy for processes or shared memory.
+
+- Run command on node 0 with memory allocated on node 0 and 1:
+
+`numactl --cpunodebind=0 --membind=0,1 {{command}}`
```

Comment:
How can I pass arguments to the command?
```suggestion
`numactl --cpunodebind={{0}} --membind={{0,1}} {{command}} {{command_arguments}}`
```

Reply:
@navarroaxel I updated your suggestion to use the advisable syntax for command arguments (`--`)